pkg/testutils/sqlutils/sql_runner.go

```diff
@@ -103,6 +103,9 @@ func (sr *SQLRunner) ExecRowsAffected(
 // matching the given regex.
 func (sr *SQLRunner) ExpectErr(t testutils.TB, errRE string, query string, args ...interface{}) {
 	t.Helper()
+	if errRE == "" {
+		t.Fatal("Empty error regular expression was provided to ExpectErr. This would assert that no error occurred. Use Exec instead, it is less confusing.")
```

Comment:
I'll need to check the call sites of this. I'm going to let CI do the first round of work for me. :D